Makefile

```diff
@@ -19,15 +19,36 @@ create-instance: ## Creates RDS Instance - mandatory: INSTANCE_NAME=[name];
 	echo -e "\nPassword Secret Endpoint:"
 	echo $(INSTANCE_NAME)-nonprod-dos_db_password
 
-download-sql-dump:
+download-sql-dump: ## Downloads the latest DoS database dump and gunzips it
 	# TODO: Downlaod the latest DoS database SQL dump file
 	# INPUT:
 	#		- link to an SQL dump file
+	make aws-s3-download URI=nhsd-texasplatform-service-dos-lk8s-nonprod/\
```

Comment:
Can we refactor this code block to something similar which first searches for the file then downloads it:
```
	file=$(shell aws s3 ls s3://nhsd-texasplatform-service-dos-lk8s-nonprod | sort | grep "dos-pg-dump-.*-clean-PU.sql.gz" | tail -n 1 | awk '{ print $$4 }')
	make aws-s3-download \
		URI=nhsd-texasplatform-service-dos-lk8s-nonprod/$$file \
		FILE=/project/build/docker/data/assets/sql/dos-dump.sql.gz
```